cmd/skywire-cli/commands/config/gen.go

```diff
@@ -222,6 +226,12 @@ var genConfigCmd = &cobra.Command{
 		if testEnv {
 			serviceConfURL = testconf
 		}
+		if autopeer {
+			ip, err := script.Exec(`ip route show | grep -i 'default via'| awk '{print $3 }'`).String()
```

Comment:
Is this even required? The hv for Skybian will always be at `192.0.0.2`, right?

Reply:
no, because if the router that the skyminer is hooked up to is at 192.168.0.1, the skyminer router cannot be the same IP address and cannot provide the same subnet as the one which it is directly connected to.

the home router and the skyminer router _can be_ the same subnet and same IP if they are separated by an intermediate router which has a different subnet as the following attempts to illustrate:
```
192.168.0.1
└── 192.168.1.1
    └── 192.168.0.1
```

without an intermediate router, many people have opted to change the subnet provided by the skyminer router to an adjacent one such as 192.168.1.1 or 192.168.2.1 if it conflicts with their home router subnet